src/crypto/sha3.cpp

```diff
@@ -10,10 +10,10 @@
 #include <span.h>
 
 #include <algorithm>
-#include <array> // For std::begin and std::end.
+#include <array>
 #include <bit>
-
-#include <stdint.h>
+#include <cstdint>
```

Comment:
3cf3695910f332f1aa8af57de7a01f63d5d7dc68: maybe change this in a separate commit?

Similar in other files, or is this auto generated?